### ISSUE TITLE
Can not open context menu in Mac OS 10.7

### ISSUE THREAD

**Issue by uk-ar** [MikeRatcliffe/Acebug#24]
I can't open editor's context menu in Mac OS 10.7(Lion).My environment is below.
- Firefox 5.01
- Firebug 1.7.2 ,1.7.1, 1.7.0

This combination works well in Windows7.
Is there a way to avoid this problem?

**Comment by nightwing** [comment]
sorry for late reply.

unfortunately i can't test onMac OS 10.7(Lion).
is this issue still present with acebug 1.8.1? and if yes do you see any errors in firefox error console
also does this happen for all ace views (html/css edit resource panel) or only large commandLine?

thanks for your help

**Comment by uk-ar** [comment]
Thank you for your reply.

I tried acebug 1.8.1 and this issue still present in
- Firebug 1.7.3, Acebug 1.8.1, Firefox 5.0.1
  There is no errors in firefox error console and this happen for only large commandLine.

And I also tried to update firefox
- Firebug 1.7.3, Acebug 1.8.1, Firefox 7.0.1
- Firebug 1.8.3, Acebug 1.8.1, Firefox 7.0.1
  Both of combinations works well in my Mac OS 10.7(Lion).
  So it seems firefox side bug...
  Thank you.
